### PR TITLE
Record initial actions by the board without a meeting

### PR DESCRIPTION
**Pull request by alice-i-cecile**
Nothing earth-shattering here, but it's good to do for transparency and others may find it interesting / helpful.

Going forward, these will be updated much more promptly after each decision: it's much easier with a structure in place.

### CODE REVIEW

File: content/foundation/meeting-minutes/actions-without-meeting/index.md
Comment (ItsDoot) [inline content/foundation/meeting-minutes/actions-without-meeting/index.md:44]:
```suggestion
![Usage of the Bevy logo by the unofficial Bevy Playground](bevy_playground_logo_usage.png)
```

**Comment by alice-i-cecile** [comment]
FYI I'm going to wait on 3 approvals by board members here to merge.

File: content/foundation/bylaws/index.md
Comment (BD103) [inline content/foundation/bylaws/index.md:104]:
```suggestion
Any action which could be taken at a meeting of the Board may be taken without a meeting if a written consent setting forth the action so taken is executed by each of the Directors entitled to vote. Such written consents may be executed in two or more counterparts, each of which shall be deemed an original and all of which, taken together, shall constitute one and the same document. Any such written consent shall be inserted in the minute book as if it were the minutes of a Board meeting. For purposes of these Bylaws, “executed” means: (a) writing that is signed; or (b) an email transmission with sufficient information to determine the sender's identity or a Discord message or vote in an official Bevy channel or private chat where all maintainers are present."
```

Just an extra space :)

File: content/foundation/meeting-minutes/actions-without-meeting/index.md
Comment (BD103) [inline content/foundation/meeting-minutes/actions-without-meeting/index.md:133]:
```suggestion
**Proposal:** Also: I think it is time to deprecate the "legacy sponsors" section. I've prepped [a branch](https://github.com/bevyengine/bevy-website/pull/1564) but I won't create the PR until we agree
```

**Review by mockersf — APPROVED** [review]

File: content/foundation/meeting-minutes/actions-without-meeting/index.md
Comment (cart) [inline content/foundation/meeting-minutes/actions-without-meeting/index.md:42]:
```suggestion
## 2024-04-18: Usage of the Bevy logo by the unofficial Bevy Playground
```

**Review by cart — APPROVED** [review]

**Pull request merged**
